### ISSUE TITLE
Cannot compile

### ISSUE THREAD

**Issue by IgorGanapolsky** [WeMakeBetterApps/MortarLib#2]
I cannot run your sample.  I get compilation errors like these:

> Error:(3, 23) error: package android.content does not exist
> Error:(4, 20) error: package android.util does not exist
> Error:(5, 22) error: package android.widget does not exist
> Error:(7, 12) error: package flow does not exist
> Error:(8, 14) error: cannot find symbol class Blueprint

**Comment by ScottPierce** [comment]
I don't have an issue running it. How would you be missing the android.content package. That's provided by the Android SDK.

Not sure what the issue could be. I can think of 2 possibilities.
1. You don't have the ANDROID_HOME environment variable set on your system. That could cause it.
2. You imported the project incorrectly. You should import the project into IntelliJ as a gradle project, and have it use the default gradle wrapper.

**Comment by IgorGanapolsky** [comment]
That's bollocks. 
a) I use Android Studio 1.2 Beta 3. 
b) If I didn't have ANDROID_HOME set, then I wouldn't be able to write Android apps.  But since I am a fulltime android developer, I write apps in android studio all day. 
c) Importing the project is as simple as File -> New -> Import Project.

**Comment by ScottPierce** [comment]
I downloaded the project directly from github, and imported it through Android Studio. Android Studio complained I was using an out of date android plugin and required that I upgrade it. I upgraded, and I was able to build the sample to an emulator.

I've pushed a new commit onto master to upgrade to the latest gradle version / android plugin. Grab master again, and let me know if that fixes your problem.
